### PR TITLE
Fix imports

### PR DESCRIPTION
**Pull request by faraonc**
Fix the imports for the new protocol buffers used for authorization.

Added download files from wget. Renamed variable `type` to `file_type` due to keyword shadowing.

### CODE REVIEW

File: hwsc-file-transaction-svc.proto [hunk 1]
```diff
@@ -1,37 +1,49 @@
...
+    }
 }
 
-message FileTransactionRequest{
-    string message = 1;
-    string name = 2;
-    string uuid = 3;
-    repeated string urls = 4;
+message FileTransactionRequest {
+    lib.Identification identification = 1;
```
Comment (dinhl) [inline hwsc-file-transaction-svc.proto]: how to use this lib.Identification identification ?

File: hwsc-file-transaction-svc.proto [hunk 2]
```diff
@@ -1,37 +1,49 @@
...
 
-message Chunk{
-    oneof test_oneof{
-        bytes buffer = 1;
-        int64 length = 2;
-        string file_name = 3;
-        string uuid = 4;
+message Chunk {
+    oneof is_one_of {
+        lib.Identification identification = 1;
```
Comment (dinhl) [inline hwsc-file-transaction-svc.proto]: also here, identification inside the oneof

Comment (faraonc) [reply hwsc-file-transaction-svc.proto]: we change the name

File: hwsc-file-transaction-svc.proto (hunk 1 above)
Comment (faraonc) [reply hwsc-file-transaction-svc.proto]: we will have a discussion about this later

**Pull request merged**